src/sentry/options/defaults.py

```diff
@@ -642,6 +642,9 @@
 register("performance.issues.render_blocking_assets.fcp_ratio_threshold", default=0.33)
 register("performance.issues.render_blocking_assets.size_threshold", default=1000000)
 
+# System-wise option for performance issue creation through issues platform
+register("performance.issues.send_to_issues_platform", default=False)
```

Comment:
You can save yourself the trouble of creating a UI for this by adding `, flags=FLAG_MODIFIABLE_BOOL` to the definition. Then you can modify it here: https://sentry.io/_admin/options/

Reply:
oh this is awesome. added it here https://github.com/getsentry/sentry/pull/45866